src/algebra/continued_fractions/computation/approximations.lean

```diff
@@ -297,18 +303,299 @@ begin
   { have : g.partial_denominators.nth n = none, by rwa seq.terminated_at at terminated,
     have : g.terminated_at n, from
       terminated_at_iff_part_denom_none.elim_right (by rwa seq.terminated_at at terminated),
-    have : (gcf.of v).denominators (n + 1) = (gcf.of v).denominators n, from
+    have : g.denominators (n + 1) = g.denominators n, from
       denominators_stable_of_terminated n.le_succ this,
     rw this },
   { obtain ⟨b, nth_part_denom_eq⟩ : ∃ b, g.partial_denominators.nth n = some b, from
       option.ne_none_iff_exists'.mp not_terminated,
     have : 1 ≤ b, from of_one_le_nth_part_denom nth_part_denom_eq,
     calc
-      (gcf.of v).denominators n ≤ b * (gcf.of v).denominators n   : by simpa using
-                                                                      mul_le_mul_of_nonneg_right
-                                                                      this zero_le_of_denom
-                            ... ≤ (gcf.of v).denominators (n + 1) : le_of_succ_nth_denom
-                                                                      nth_part_denom_eq }
+      g.denominators n ≤ b * g.denominators n   : by simpa using (mul_le_mul_of_nonneg_right
+                                                                 this zero_le_of_denom)
+                   ... ≤ g.denominators (n + 1) : le_of_succ_nth_denom nth_part_denom_eq }
+end
+
+/-
+Next we prove the so-called *determinant formula* for `gcf.of`:
+`Aₙ * Bₙ₊₁ - Bₙ * Aₙ₊₁ = (-1)^(n + 1)`.
+-/
+section determinant
+
+lemma determinant_aux (hyp: n = 0 ∨ ¬(gcf.of v).terminated_at (n - 1)) :
+  let g := gcf.of v in
```

Comment:
`let` statements aren't completely transparent in lemma statements. So this might make it harder to use later on. But since this is an `_aux` lemma, I'm assuming it will only be used in a proof below, and it works out fine there.
So feel free to do nothing with this remark.